TUTORIAL_FOR_BEGINNERS.md

```diff
@@ -0,0 +1,200 @@
+# English Version
+---
+
+## Preparing to contribute with the Querido Diário’s scrapers
+
+The [Querido Diário’s scrapers][raspQD] are made on [Python 3] with [Scrapy] lib and it’s hosted on [GitHub]. This guide assume that you’re starting your contact with Python e you are already have Python 3 installed, hence we’re going to introduce you what you need to prepare yourself for the sprint: GitHub, git, dependencies set up and a guide showing you how to analyses websites with scrapers.
+
+### Summary
+
+#### - <a href="#win-en">Step by step on Windows</a>
+#### - <a href="#linux-en">Step by step on Linux</a>
+
+##### <ins><div id="win-en">Step by step on Windows</div></ins>
+
+1. The contribution platform is GitHub, so let’s [create an account][account].
+
+2. Access the [Querido Diário’s scrapers official repository][repQD]
+
+3. Create your fork from the project. [Here][tutoGitHub] there is a contribution tutorial on GitHub (Read until step 01 “How to open a Pull Request?”).
+
+4. Let’s download and install [VSCode] on your computer (you can follow the pattern installation).
+    - The VSCode isn’t mandatory, any text editor works, but to simplify, during the sprint we’ll assume that you’re using the terminal with PowerShell on VSCode.
+
+5. After installing VSCode, let’s install the [git]. It’s important to check these 2 options:
+
+![choosingEditorGit](assets/img01.png)
+
+![pathEnvinment](assets/img02.png)
+
+6. Open VSCode and open the folder where you want the project folder be located.
+
+![openFolder](assets/img03.png)
+
+7. Now, we must open the terminal:
+
+![terminal](assets/img04.png)
+
+8. Let’s clone your fork with an opened terminal. [This git tutorial][tutoGit] show how to do it (Read until the step “It’s possible copy more than one repository to my local repository?”)
+
+9. Now you’ll need to install the [Microsoft Visual Build Tools]:
+
+    ![mvbt](assets/img05.png)
+
+    - When you open it, you have to choose the option “Development for desktop C++” and check if there are some additional contents checked as Windows 10 SDK and the build tools from MSVC v142.
+
+    ![winSDK](assets/img06.png)
+
+    - When you’ve already checked everything, carry on and wait to download the necessary files. After downloading the Visual Studio, it’s time to prepare the virtual environment to start.
```

Comment:
Do you think the images are a good addition? Or just the text with would be enough?

Reply:
Putting myself in a beginner's shoes I think it could be helpful since the Querido Diário's project may be someone's first open source project on GitHub.